frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,441 @@
+                ></div>
+                <p class="ml-20 text-white font-bold text-xl">JourneyPlanner</p>
+                <div class="w-full flex justify-end h-full items-center">
+                  <SvgAirplaneIcon class="w-9 mr-5 pb-1" />
+                </div>
+              </div>
+              <div class="flex h-5/6">
+                <div
+                  class="h-full w-full rounded-b-2xl bg-background border-border-gray border-x-2 border-b-2 flex justify-center dark:bg-border-dark dark:border-form-input-dark"
+                >
+                  <div class="h-full w-full flex flex-col items-end relative">
+                    <img
+                      class="absolute right-[50%] top-[25%] z-20 translate-x-[50%] -translate-y-[25%] w-2/5"
+                      :src="qrcode"
+                      alt="QR Code"
+                    />
+                    <div
+                      class="absolute ml-10 rounded-full border-input-placeholder text-input-placeholder w-16 h-16 self-center border-dashed border-2 right-2 bottom-2 flex text-center justify-center text-xs pl-1.5 pr-1.5 pt-1 z-40 dark:border-white dark:text-white"
+                    >
+                      <T keyName="journey.turn" />
+                    </div>
+                    <SvgStripes class="z-0 lg:w-1/2 md:w-2/3" />
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        class="ml-[10%] lg:w-1/3 md:w-[50%] md:visible invisible w-0 max-md:h-0"
+      >
+        <div
+          class="h-1/6 bg-border border-x-2 border-t-2 border-border-darker rounded-t-2xl flex items-center relative dark:bg-ticket-top-dark-bg dark:border-border-blue-dark"
+        >
+          <div
+            class="absolute ml-10 rounded-full w-7 h-7 bg-border-gray inline-block self-center"
+          ></div>
+          <p class="ml-20 text-white font-bold text-xl">JourneyPlanner</p>
+          <div class="w-full flex justify-end h-full items-center">
+            <SvgAirplaneIcon class="w-9 mr-5" />
+          </div>
+        </div>
+        <div class="flex h-5/6">
+          <div
+            class="w-full rounded-b-2xl bg-background border-border-gray border-l-2 border-b-2 dark:bg-border-dark dark:border-form-input-dark"
+          >
+            <div class="w-full grid grid-cols-4 relative">
+              <div
+                class="w-full col-span-3 pl-10 flex flex-col h-[120%] justify-center font-medium"
+              >
+                <T keyName="form.input.journey.name" />
+                <input
+                  class="w-full rounded-md px-2.5 pb-1 mb-2 pt-1 text-md text-text dark:text-white font-bold bg-input-gray focus:outline-none focus:ring-1 dark:bg-input-dark"
+                  disabled
+                  :value="journeyData.name"
+                />
+                <T keyName="form.input.journey.destination" class="" />
```

Comment:
remove unused class, it even throws a warning

![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/75267822/2dc5d86f-515b-459b-91a1-75690ad196bc)